tests/test_collections.py

```diff
@@ -241,31 +241,6 @@ def test_collection_slicing(fc):
     assert type(fc[:]) is FeatureCollection
 
 
-@mock.patch('telluric.collections.rasterize')
```

Comment:
I removed these tests because:

> The mock cannot be done anymore, since now the module
is imported inside the function to avoid circular
import errors. On the other hand, this tests is only
testing some calling conventions, and there are plenty
of other tests that test the actual result of the
rasterization.